src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1465,6 +1465,12 @@
       (format "ALTER TABLE reports ADD COLUMN logs %s DEFAULT NULL" jsonb-type)
       (format "ALTER TABLE reports ADD COLUMN resources %s DEFAULT NULL" jsonb-type))))
 
+(defn factset-hash-field-not-nullable
+  []
+  (jdbc/do-commands
+   "UPDATE factsets SET hash='' WHERE hash is NULL"
```

Comment:
I don't think we're using hashes for that, otherwise we'd have problems when we change hash fields too.